packages/core/android/src/main/kotlin/com/datadog/reactnative/DdTraceImplementation.kt

```diff
@@ -43,6 +43,10 @@ class DdTraceImplementation(
         val span = tracer.buildSpan(operation)
             .withStartTimestamp(TimeUnit.MILLISECONDS.toMicros(timestampMs.toLong()))
             .start()
+        
+        // This is required for traces to be able to be bundled with logs.
+        tracer.scopeManager().activate(span)
```

Comment:
This part was missing in the original PR, but it seems strange to me that this is needed for traces to be bundled with logs.

Reply:
That is because the only way to get information about trace context in the subscription manner is to listen for scope activated/scope closed events https://github.com/DataDog/dd-sdk-android/blob/9c75335eec3fc3ef5f5dded88060cf3ac64ab288/features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/AndroidTracer.kt#L48 (span creation doesn't activate the scope automatically).

Also I guess the missing part here is to close the scope once span is finished.